tests/rt_gnu.conf

```diff
@@ -56,6 +60,7 @@ RUN     | control_thompson_progcld_thompson_debug
 RUN     | control_ras_debug                                                                                                       |                | fv3         |
 RUN     | control_stochy_debug                                                                                                    |                | fv3         |
 RUN     | control_debug_p8                                                                                                        |                | fv3         |
+RUN     | control_debug_p8_gf_mynn_hfip                                                                                           |                |             |
```

Comment:
This line should have "fv3" in it so it will generate a baseline. Also, you need to update the debug compile line to compile this suite.